Makefile

```
@@ -22,8 +30,8 @@ verify: fmt vet lint
 # Run tests
 test: generate verify manifests unit
 
-unit:
-	hack/unit-tests.sh
+unit: envtest
+	PATH=$(PATH):$(KUBEBUILDER_ASSETS) go test ./... -coverprofile cover.out
```

It looks like to override the default binaries path (`/usr/local/kubebuilder/bin`) we need to set the `KUBEBUILDER_ASSETS` env var rather than adding its value to the `PATH` env.
ref: https://github.com/kubernetes-sigs/controller-runtime/blob/11917e9d90625b51e5dab16b4cf20934971e2f08/pkg/envtest/server.go#L44
```suggestion
        PATH="$(PATH)" KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test ./... -coverprofile cover.out
```